lpd/models.py

```diff
@@ -154,6 +154,9 @@ def update_scores(cls, learner):
         Updates scores for knowledge components representing groups
         that a given learner might belong to,
         based on all of the learner's QualitativeAnswers.
+
+        Scores need to be equal to (1 - probability) to have
+        the desired effect on recommendations generated by the adaptive engine.
```

Comment:
@tomaszgy Very nice, thanks for adding this 😀

Reply:
😃